### PR TITLE
removed ES5 shim

### PR DESCRIPTION
**Pull request by NayanKhedkar**

### CODE REVIEW

File: src/core/js/scriptLoader.js [hunk 1]
```diff
@@ -101,7 +75,7 @@
         ], loadAdapt);
     }
 
-    //7. Load adapt
+    //6. Load adapt
     function loadAdapt() {
         switch (IE) {
```
Comment (moloko) [inline src/core/js/scriptLoader.js]: it looks like you could also get rid of this `switch` statement as well

File: src/core/js/scriptLoader.js [hunk 2]
```diff
@@ -101,18 +48,12 @@
...
-        case 8: case 9:
-            //ie8 and ie9 don't do crossdomain with jquery normally
-            break;
-        default:
-            //cross domain support for all other browers
-            $.ajaxPrefilter(function( options ) {
-                options.crossDomain = true;
-            });
-        }
+        //cross domain support for all other browers
```
Comment (moloko) [inline src/core/js/scriptLoader.js]: comment no longer necessary

**Comment by moloko** [comment]
@tomgreenfield I'm happy to leave as is

**Comment by oliverfoster** [comment]
me too, unless it's going to make life a lot easier in the future?

**Comment by sarveshwar-gavhane** [comment]
@NayanKhedkar  would you mind removing references ? https://github.com/adaptlearning/adapt_framework/blob/master/grunt/config/watch.js#L32
https://github.com/adaptlearning/adapt_framework/blob/master/grunt/config/watch.js#L83

**Comment by oliverfoster** [comment]
conflict needs resolving here

**Comment by NayanKhedkar** [comment]
conflict resolved,Please refer the below PR.
PR: https://github.com/adaptlearning/adapt_framework/pull/1762

**Pull request closed without merging**